File: Laboratorio/Lab09/Practica 8/Alumno.java


public class Alumno{
	private String nombre;
	int edad;

	public Alumno(){
		nombre = "nadie";
		edad = 0;
	}

	public Alumno(String nombre, int edad){
		this.nombre = nombre;
		this.edad = edad;
	}

	public void setNombre(String nombre){
		this.nombre = nombre;
	}

	public String getNombre(){
		return this.nombre;
	}

	public String toString(){
		return "Yo me llamo " + nombre + " y mi edad es: " + edad;
	}

	void leer(){
		System.out.println("...Leyendo...");
	}

	void escribir(){
		System.out.println("...Escribiendo...");
	}

}

File: Laboratorio/Lab09/Practica 8/Alumno.py
class Alumno():
	nombre=""
	edad = 0

	def __init__(self):
		self.nombre = "nobody"
		self.edad = 0

	def leer(self):
		print("...leyendo...")

	def escribir(self):
		print("...escribiendo...")

dormilon = Alumno()
despierto = Alumno()
dormilon.leer()
despierto.escribir()
print("yo soy " , dormilon.nombre)